### PR TITLE
switch to python3

### PR DESCRIPTION
**Pull request by mr-c**
Adapted from https://salsa.debian.org/med-team/vg/-/blob/a5a7bde0b1c3c0a02ed6559cb26989e0ad40df01/debian/patches/python3

### CODE REVIEW

**Comment by mr-c** [comment]
This passes both Travis CI and GitLab CI, but one of the Travis CI runs is not reporting back to GitHub..

File: scripts/filter-noisy-assembler-warnings.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```
Comment (adamnovak) [inline scripts/filter-noisy-assembler-warnings.py:1]: Shouldn't these all be `/usr/bin/env python3`?

**Comment by adamnovak** [comment]
I'll merge this up, we can change the interpreter path if someone complains it isn't working right in a virtualenv.

File: scripts/filter-noisy-assembler-warnings.py (hunk 1 above)
Comment (mr-c) [reply scripts/filter-noisy-assembler-warnings.py:1]: Yes, sorry. This patch mixed two objectives: py3 + absolute paths (for Debian). I should have kept the 2nd part out

**Pull request merged**